packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/functionService.test.ts

```diff
@@ -327,4 +338,130 @@ describe('Function Service', () => {
       expect(fsSyncStub.callCount).to.equal(2);
     });
   });
+
+  describe('Debug Configuration', () => {
+    let fsSyncStub: SinonStub;
+
+    beforeEach(() => {
+      fsSyncStub = sandbox.stub(fs, 'existsSync');
+    });
+
+    it('Should set remoteRoot to /workspace when JavaScript and running in a container.', () => {
+      const functionDef: FunctionExecution = {
+        rootDir: 'FirstFunction',
+        debugPort: 7777,
+        port: 8080,
+        debugType: 'unknown',
+        terminate: () => Promise.resolve(),
+        isContainerLess: false
+      };
+
+      fsSyncStub.onCall(0).returns(false);
+      fsSyncStub.onCall(1).returns(true);
```

Comment:
unit test approach question.  Why not stub `getFunctionType` in these tests to make it a bit more clear how we're getting JS vs TS vs Java function in the function under test?

Reply:
That was the existing pattern in the file.

Reply:
done